.github/workflows/testcases.yml

```diff
@@ -0,0 +1,22 @@
+name: running testCases
```

Comment:
I would recommend adding in maven code convension validation. I did the same in sample calculator